src/test/resources/pages/instructorCourseEditTestingSanitization.html

```diff
@@ -42,7 +42,7 @@ <h1>
             Course Name:
           </label>
           <div class="col-sm-9">
-            <input class="form-control" data-original-title="The name of the course, e.g. Software Engineering." data-placement="top" data-toggle="tooltip" disabled="" id="coursename" maxlength="64" name="coursename" tabindex="2" title="" type="text" value="Testing<script> alert('hi!'); </script>">
+            <input class="form-control" data-original-title="The name of the course, e.g. Software Engineering." data-placement="top" data-toggle="tooltip" disabled="" id="coursename" maxlength="64" name="coursename" tabindex="2" title="" type="text" value="Testing&lt;script&gt; alert(&#39;hi!&#39;); &lt;&#x2f;script&gt;">
```

Comment:
This does not look intended either.

Reply:
@wkurniawan07 For unintended change that happens due to GodMode earlier, what should I do ? As changing it back to its original will cause the step to fail

Reply:
This change has nothing to do with the issue at hand.

This is definitely caused by one of the lines in which you added `SanitizationHelper.sanitizeForHtml(name)`, which I have requested to undo. After you do that this change should be gone as well.